Azure Pipelines e2e tests are failing

### Report

Test output:
```
 >>> scalers/azure-pipelines.test.ts.log <<<

  Uncaught exception in scalers/azure-pipelines.test.ts

  scalers/azure-pipelines.test.ts(***,24): error TS2307: Cannot find module 'azure-devops-node-api' or its corresponding type declarations.
  scalers/azure-pipelines.test.ts(2,2***): error TS2307: Cannot find module 'azure-devops-node-api/BuildApi' or its corresponding type declarations.
  scalers/azure-pipelines.test.ts(3,2***): error TS2307: Cannot find module 'azure-devops-node-api/TaskAgentApiBase' or its corresponding type declarations.
  scalers/azure-pipelines.test.ts(4,2***): error TS2307: Cannot find module 'azure-devops-node-api/interfaces/TaskAgentInterfaces' or its corresponding type declarations.

  ✖ scalers/azure-pipelines.test.ts exited with a non-zero exit code: ***

  *** uncaught exception
```

--- a/tests/package.json
+++ b/tests/package.json
@@ -22,6 +22,7 @@
     "async": "^3.1.0",
     "ava": "^2.4.0",
     "axios": "^0.21.1",
+    "azure-devops-node-api": "^10.2.2",
     "azure-storage": "^2.10.3",
     "chalk": "^2.4.2",
     "eol": "^0.9.1",

